From b9f02989310a3c920589f1aa646bcbf16b2572a0 Mon Sep 17 00:00:00 2001
From: Ross Burton <ross.burton@intel.com>
Date: Wed, 13 Jun 2012 11:07:33 +0100
Subject: [PATCH] Inherit the GNOME icon theme

If the GNOME icon theme isn't installed this is a no-op, but if it is then
you'll get fallback from Sato to GNOME for icons that are not in Sato.

As applications that use the full GNOME icon theme depend on gnome-icon-theme,
this just works.

Upstream-Status: Backport
---
 index.theme |    1 +
 1 file changed, 1 insertion(+)

diff --git a/index.theme b/index.theme
index c4b03a7..688fbbf 100644
--- a/index.theme
+++ b/index.theme
@@ -1,6 +1,7 @@
 [Icon Theme]
 Name=Sato
 Comment=Sato icon theme
+Inherits=Adwaita
 
 Directories=16x16/apps,16x16/status,16x16/stock,16x16/places,16x16/mimetypes,16x16/devices,16x16/actions,22x22/apps,22x22/status,22x22/stock,22x22/places,22x22/mimetypes,22x22/devices,22x22/actions,32x32/apps,32x32/status,32x32/stock,32x32/places,32x32/mimetypes,32x32/devices,32x32/actions,48x48/apps,48x48/status,48x48/stock,48x48/places,48x48/mimetypes,48x48/devices,48x48/actions,64x64/apps,64x64/status,64x64/stock,64x64/places,64x64/mimetypes,64x64/devices,64x64/actions,
 
-- 
1.7.10

